easybuild/easyconfigs/l/LAST/LAST-869-intel-2017a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'MakeCp'
+
+name = 'LAST'
+version = '869'
+
+homepage = 'http://last.cbrc.jp/'
+description = """ LAST finds similar regions between sequences. The main technical innovation
+ is that LAST finds initial matches based on their multiplicity,
+ instead of using a fixed length (e.g. BLAST uses 11-mers)."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'cstd': 'c++11', 'opt': True}
+
+source_urls = ['http://last.cbrc.jp/']
+sources = ['%(namelower)s-%(version)s.zip']
+
+patches = ['%(name)s-%(version)s_makefile.patch']
```

Comment:
There should be no need for a patch, in the existing easyconfig for `LAST` we use:

```python
buildopts = 'CC="$CC" CXX="$CXX" CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS -pthread -DHAS_CXX_THREADS"'

Reply:
@boegel: `CC`, `CXX`, `CFLAGS`, and `CXXFLAGS` hard encoded in `last-869/src/makefile`, lines 1-9:
```
CXX = g++
CC  = gcc

CXXFLAGS = -O3 -Wall -Wextra -Wcast-qual -Wswitch-enum -Wundef	\
-Wcast-align -pedantic -g -std=c++11 -pthread -DHAS_CXX_THREADS
# -Wconversion
# -fomit-frame-pointer ?

CFLAGS = -Wall -O2
```

Reply:
Even with a `makefile` like this, you can override values by passing arguments to `make`, like we do in the existing `LAST` easyconfig...

Reply:
@boegel Done